package/kedro_viz/models/graph.py

```diff
@@ -51,6 +50,10 @@ def _pretty_name(name: str) -> str:
     return " ".join(parts)
 
 
+def _strip_namespace(name: str) -> str:

```

Comment:
I am no python developer, but I noticed that the namespace issue does not apply to task and parameter nodes, which might suggest that this has been implemented already somewhere in the code which suggests that this might not be needed?

Reply:
The namespace is added explicitly to dataset's name on the Kedro side. It doesn't get added to node's. 

@rashidakanchwala have you checked parameters?